### PR TITLE
feat: Shuffle all fields on a line

### PR DESCRIPTION
**Pull request by sharma-shruti**

### CODE REVIEW

File: src/shuffle_fields_on_each_line.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+
+# Shuffle all fields on each line
+
+*Pomodoro: 4*
+
+In this one-liner, the code to execute inside of the quotation marks is shuffle @F, which shuffles the fields and returns the shuffled list.
+
```
Comment (abhi18av) [inline src/shuffle_fields_on_each_line.md:7]: In this position please add the sample file contents. This way the project would be reproducible.

I recognize that you are using the `standard input` i.e. the `stdin` for these examples. But let's try to stick to a file based setup.

**Review by abhi18av — APPROVED** [review]
Approving - with a suggestion.

File: src/shuffle_fields_on_each_line.md (hunk 1 above)
Comment (sharma-shruti) [reply src/shuffle_fields_on_each_line.md:7]: Ok. The Perl example field here shows the exact way how they have written the examples in the book. I try to re-create similar input using Powershell. But you may have noticed, I've also tried doing the same thing using a file content. 

```powershell
PS /root> Get-Content ./word.txt                          
a b c d

PS /root> cat ./word.txt | %{$_.Split(" ")[1..3] + $_[0] }
b
c
d
a
```

Comment (abhi18av) [reply src/shuffle_fields_on_each_line.md:7]: Oh, yes. I missed it earlier. 

Anyways, what I meant is that  on the terminal whenever you use `<` or `>` or `|` that's `stdin` or `stdout`. 

I'm requesting you to please add the a section to the markdown which contains the contents of the file. A minimal file is fine

Comment (sharma-shruti) [reply src/shuffle_fields_on_each_line.md:7]: ok, got it

**Comment by abhi18av** [comment]
@sharma-shruti , I'm merging this one. Please keep in mind the `file` section for future PRs.

**Pull request merged**